src/Compilers/CSharp/Test/Emit/Emit/DesktopStrongNameProviderTests.cs

```diff
@@ -21,14 +21,14 @@ public void RespectCustomTempPath()
         {
             var tempDir = Temp.CreateDirectory();
             var provider = new DesktopStrongNameProvider(tempPath: tempDir.Path);
-            Assert.Equal(tempDir.Path, provider.FileSystem.GetTempPath());
+            Assert.Equal(tempDir.Path, provider.FileSystem.GetSigningTempPath());
         }
 
         [Fact]
-        public void RespectDefaultTempPath()
+        public void RespectEmptyTempPath()
```

Comment:
I didn't understand how the new name of the test reflects the new assert.

Reply:
Changed to `RespectNullTempPath`